<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmJanelaPrincipal</class>
 <widget class="QMainWindow" name="frmJanelaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>731</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sibi</string>
  </property>
  <property name="windowIcon">
   <iconset resource="arquivos.qrc">
    <normaloff>:/Arquivos/Imagens/PUC.png</normaloff>:/Arquivos/Imagens/PUC.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>802</width>
      <height>620</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="6" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QLabel" name="codigoDoExemplarLabel_Devolucao">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>96</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>Codigo Do Exemplar</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="codigoDoExemplarLineEdit_Devolucao">
         <property name="maximumSize">
          <size>
           <width>87</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_OK_Devolucao">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>OK</string>
         </property>
         <property name="icon">
          <iconset resource="arquivos.qrc">
           <normaloff>:/Arquivos/Imagens/OK.png</normaloff>:/Arquivos/Imagens/OK.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_2">
         <property name="minimumSize">
          <size>
           <width>231</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 0px solid grey;
text-align: left; 
</string>
         </property>
         <property name="text">
          <string>DEVOLUÇÃO</string>
         </property>
         <property name="icon">
          <iconset resource="arquivos.qrc">
           <normaloff>:/Arquivos/Imagens/chevron.png</normaloff>:/Arquivos/Imagens/chevron.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_Reserva">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px solid grey;
text-align: left; 
</string>
       </property>
       <property name="text">
        <string>RESERVA</string>
       </property>
       <property name="icon">
        <iconset resource="arquivos.qrc">
         <normaloff>:/Arquivos/Imagens/chevron.png</normaloff>:/Arquivos/Imagens/chevron.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="2" rowspan="8">
      <widget class="Line" name="line_2">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="Line" name="line_3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0" rowspan="2" colspan="2">
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <widget class="QLineEdit" name="codigoDoExemplarLineEdit">
         <property name="minimumSize">
          <size>
           <width>85</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="cPFLineEdit">
         <property name="minimumSize">
          <size>
           <width>85</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="codigoDoExemplarLabel">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>140</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>Codigo Do Exemplar</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="cPFLabel">
         <property name="maximumSize">
          <size>
           <width>96</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>CPF</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_OK">
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>OK</string>
         </property>
         <property name="icon">
          <iconset resource="arquivos.qrc">
           <normaloff>:/Arquivos/Imagens/OK.png</normaloff>:/Arquivos/Imagens/OK.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>231</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>231</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 0px solid grey;
text-align: left; 
</string>
         </property>
         <property name="text">
          <string>EMPRESTIMO</string>
         </property>
         <property name="icon">
          <iconset resource="arquivos.qrc">
           <normaloff>:/Arquivos/Imagens/chevron.png</normaloff>:/Arquivos/Imagens/chevron.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="3" rowspan="8">
      <widget class="frmConsultaAcervo" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>510</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>54645</width>
         <height>5645</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="label_layout">
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/Arquivos/Imagens/sibi-puc-goias.gif);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="frmReserva" name="widget_Reserva" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>150</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="Line" name="line_4">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuUsuario">
    <property name="title">
     <string>Usuario</string>
    </property>
    <addaction name="actionCadastrar"/>
    <addaction name="actionConsulta"/>
   </widget>
   <widget class="QMenu" name="menuAcervo">
    <property name="title">
     <string>Acervo</string>
    </property>
    <widget class="QMenu" name="menuCadastrar">
     <property name="title">
      <string>Cadastrar</string>
     </property>
     <addaction name="actionLivro"/>
     <addaction name="actionRevista"/>
     <addaction name="actionArtigo"/>
     <addaction name="actionVideo"/>
    </widget>
    <widget class="QMenu" name="menuConsulta">
     <property name="title">
      <string>Consulta</string>
     </property>
     <addaction name="actionLivro_2"/>
     <addaction name="actionRevista_2"/>
     <addaction name="actionArtigo_2"/>
     <addaction name="actionVideo_2"/>
     <addaction name="actionExemplar"/>
    </widget>
    <addaction name="menuCadastrar"/>
    <addaction name="menuConsulta"/>
   </widget>
   <widget class="QMenu" name="menuEditoras">
    <property name="title">
     <string>Editoras</string>
    </property>
    <addaction name="actionCadastro"/>
    <addaction name="actionConsulta_2"/>
   </widget>
   <widget class="QMenu" name="menuAutores">
    <property name="title">
     <string>Autores</string>
    </property>
    <addaction name="actionCadastro_2"/>
    <addaction name="actionConsulta_3"/>
   </widget>
   <addaction name="menuUsuario"/>
   <addaction name="menuAcervo"/>
   <addaction name="menuEditoras"/>
   <addaction name="menuAutores"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCadastrar">
   <property name="icon">
    <iconset resource="arquivos.qrc">
     <normaloff>:/Arquivos/Imagens/cadastro.png</normaloff>:/Arquivos/Imagens/cadastro.png</iconset>
   </property>
   <property name="text">
    <string>Cadastrar</string>
   </property>
  </action>
  <action name="actionConsulta">
   <property name="icon">
    <iconset resource="arquivos.qrc">
     <normaloff>:/Arquivos/Imagens/listar.png</normaloff>:/Arquivos/Imagens/listar.png</iconset>
   </property>
   <property name="text">
    <string>Consulta</string>
   </property>
  </action>
  <action name="actionLivro">
   <property name="text">
    <string>Livro</string>
   </property>
  </action>
  <action name="actionRevista">
   <property name="text">
    <string>Revista</string>
   </property>
  </action>
  <action name="actionArtigo">
   <property name="text">
    <string>Artigo</string>
   </property>
  </action>
  <action name="actionVideo">
   <property name="text">
    <string>Video</string>
   </property>
  </action>
  <action name="actionLivro_2">
   <property name="text">
    <string>Livro</string>
   </property>
  </action>
  <action name="actionRevista_2">
   <property name="text">
    <string>Revista</string>
   </property>
  </action>
  <action name="actionArtigo_2">
   <property name="text">
    <string>Artigo</string>
   </property>
  </action>
  <action name="actionVideo_2">
   <property name="text">
    <string>Video</string>
   </property>
  </action>
  <action name="actionCadastro">
   <property name="text">
    <string>Cadastro</string>
   </property>
  </action>
  <action name="actionConsulta_2">
   <property name="text">
    <string>Consulta</string>
   </property>
  </action>
  <action name="actionCadastro_2">
   <property name="text">
    <string>Cadastro</string>
   </property>
  </action>
  <action name="actionConsulta_3">
   <property name="text">
    <string>Consulta</string>
   </property>
  </action>
  <action name="actionExemplar">
   <property name="text">
    <string>Exemplar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>frmConsultaAcervo</class>
   <extends>QWidget</extends>
   <header>frmConsultaAcervo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>frmReserva</class>
   <extends>QWidget</extends>
   <header>frmReserva.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="arquivos.qrc"/>
 </resources>
 <connections/>
</ui>
